renderer/brave_content_renderer_client.cc

```diff
@@ -8,8 +9,8 @@
 BraveContentRendererClient::BraveContentRendererClient()
     : ChromeContentRendererClient() {}
 
-void BraveContentRendererClient::SetRuntimeFeaturesDefaultsBeforeBlinkInitialization() {
-  blink::WebRuntimeFeatures::EnableWebUsb(false);
+void BraveContentRendererClient::
+SetRuntimeFeaturesDefaultsBeforeBlinkInitialization() {
   blink::WebRuntimeFeatures::EnableSharedArrayBuffer(false);
```

Comment:
Not your issue, but shouldn't we be calling the base class here?


```
void ChromeContentRendererClient::
    SetRuntimeFeaturesDefaultsBeforeBlinkInitialization() {
  // The performance manager service interfaces are provided by the chrome
  // embedder only.
  blink::WebRuntimeFeatures::EnablePerformanceManagerInstrumentation(true);

// Web Share is shipped on Android, experimental otherwise. It is enabled here,
// in chrome/, to avoid it being made available in other clients of content/
// that do not have a Web Share Mojo implementation.
#if defined(OS_ANDROID)
  blink::WebRuntimeFeatures::EnableWebShare(true);
  blink::WebRuntimeFeatures::EnableWebShareV2(true);
#endif

  if (base::FeatureList::IsEnabled(subresource_filter::kAdTagging))
    blink::WebRuntimeFeatures::EnableAdTagging(true);
}
```

Reply:
@bbondy  Yup, we should call base class' method also if we don't want prevent all.
I assume that when @jumde pushed https://github.com/brave/brave-core/pull/114, base class methods would be like below.
```
void ChromeContentRendererClient::
    SetRuntimeFeaturesDefaultsBeforeBlinkInitialization() {
// Web Share is shipped on Android, experimental otherwise. It is enabled here,
// in chrome/, to avoid it being made available in other clients of content/
// that do not have a Web Share Mojo implementation.
#if defined(OS_ANDROID)
  blink::WebRuntimeFeatures::EnableWebShare(true);
#endif
}
```
So, @pj might be skipped calling it.
Added.

Reply:
I think maybe it is safest to always call it and just undo anything we don't want to do. But probably there is nothing else we don't want to do.  We'll have android support soon btw.